### DISCUSSION TITLE
Attack Map location change?

### DISCUSSION THREAD

**Discussion by J4yB33** [telekom-security/tpotce#1070] [Q&A] [answered]
Hello all, 

I have been using Tpot for a while - it is fantastic!
I've just redeployed the very latest and the Attack map is just stunning!

My question is, can I change the location from germany to my location (UK)?
![Screenshot 2022-04-13 at 20 01 56](https://user-images.githubusercontent.com/34707278/163251570-bd26f10d-d8b4-4167-a7f7-e1c6a15647a6.png)

Many thanks in advance

JB

**Comment by t3chn0m4g3** [comment, marked as answer]
By default T-Pot will retrieve your public IP and use the GeoIP info for the Attack Map. If it is NIL or cannot be retrieved the info it will fallback to the birthplace of T-Pot 😇
You can adjust this in `bin/updateip.sh`.

**Comment by dmnitro** [comment]
Hey i was wondering what software or VM you use to use Tpotce ive been trying to download it for the past 3 days and i can never access the web interface today the closest i got ever was asking for credentials but didn't do anything when i pressed log in please let me know would appreciate it a lot

**Reply by t3chn0m4g3** [reply]
Recommend the Readme, it is all documented. Also this question is off topic.
